tutorials/script/core/7_pre_response_processing.py

```diff
@@ -97,10 +97,11 @@ async def modified_response(self, original_response, ctx):
             except Exception as exc:
                 return str(exc)
 
+However, this functionality is now available in the core library
+as the FallbackResponse class.
```

Comment:
Link to the class API ref using [doclink](https://github.com/deeppavlov/chatsky/blob/master/docs/source/utils/notebook.py#L153). Examples of usages of `mddoclink` can be found in this (7th pre-response) tutorial.